docs/registry/getting-started-registry/quickstart.yml

```diff
@@ -5,6 +5,10 @@ metadata:
   annotations:
     draft: false
     order: 1
+  labels:
+    service-category/application-services: true
+    product/rhosr: true
+    technology: quarkus
```

Comment:
I think the quarkus entry is not needed here?

Reply:
removed ref to 
  technology: quarkus